version/rawversion/version.go

```diff
@@ -1,10 +1,10 @@
 package rawversion
-
+61;7604;1c61;7604;1c
```

Comment:
line is added in both the release bump and the -dev bump so you need to remove it from both commits